src/plugins/api/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "api",
```

Comment:
Agreed, I'll move it into the kibana plugin.

And yes, Marvel currently exposes api endpoints as `/marvel/api`